Assets/_Source/Hathora/ClientManager.cs

```diff
@@ -0,0 +1,170 @@
+using UnityEngine;
```

Comment:
I renamed this file from Client to ClientManager, as Client is a name used by the library. Git got a little confused by this.